invenio_base/cli.py

```diff
@@ -80,6 +82,25 @@ def list_entrypoints(entry_point):
             click.echo('  {0}'.format(ep))
 
 
+@instance.command('migrate_secret_key')
+@click.option('--old-key', required=True)
+@with_appcontext
+def migrate_secret_key(old_key):
+    """Call entry points exposed for the SECRET_KEY change."""
+    if 'SECRET_KEY' not in current_app.config:
+        click.secho('SECRET_KEY is not set in the configuration.', fg='red')
+        return
+
+    for ep in iter_entry_points('invenio_base.secret_key'):
+        try:
+            ep.load()(current_app._get_current_object(), old_key=old_key)
```

Comment:
This code is not sending a signal anymore. No need for `current_app._get_current_object()`